insights/specs/default.py

```diff
@@ -618,6 +618,7 @@ def tomcat_home_base(broker):
     vgs = None  # simple_command('/sbin/vgs -v -o +vg_mda_count,vg_mda_free,vg_mda_size,vg_mda_used_count,vg_tags --config="global{locking_type=0}"')
     vgs_noheadings = simple_command("/sbin/vgs --nameprefixes --noheadings --separator='|' -a -o vg_all --config=\"global{locking_type=0}\"")
     vgs_noheadings_all = simple_command("/sbin/vgs --nameprefixes --noheadings --separator='|' -a -o vg_all --config='global{locking_type=0} devices{filter=[\"a|.*|\"]}'")
+    virsh_list_all = simple_command("/usr/bin/virsh --readonly list --all")
```

Comment:
For new `simple_command`, please add its corresponding `simple_file` to the `insights_archives.py` as well